src/GNDStk/type2node/test/type2node.test.cpp

```diff
@@ -19,10 +19,16 @@ SCENARIO("Testing GNDStk convert(type,node)") {
    // So, we'll just have type be the sequence containers in our built-in
    // convert(type,node) function.
 
+   // Update, 2020-10-19. The relevant convert() functionality has been
+   // reformulated into the detail::convert_pcdata_text_t callable object,
+   // so we'll reformulate with that. Ultimately, due to the location of
+   // the new code, we should probably put this test material elsewhere.
+
    WHEN("type is vector") {
       std::vector<double> container = { 1, 2.3, 4.56 };
       node n("old name");
-      convert(container,n);
+      // convert(container,n);
```

Comment:
If you want to have such a comment like this, then you could (not necessarily should) put it in a 

```
THEN( "..." ){
  ...
}
```

block.

Reply:
I've just gone ahead and removed those.